earn/src/components/boost/ImportModal.tsx

```diff
@@ -175,77 +259,147 @@ export default function ImportModal(props: ImportModalProps) {
-            <LeverageSlider
-              type='range'
-              list='boost-factor-labels'
-              min={BOOST_MIN}
-              max={BOOST_MAX}
-              step={1}
-              value={boostFactor}
-              onChange={(e) => setBoostFactor(parseInt(e.target.value) || 0)}
-            />
-            <StyledDatalist id='boost-factor-labels'>
-              {labels.map((label, i) => (
-                <option key={i} value={i + 1} label={label}></option>
-              ))}
-            </StyledDatalist>
-            <Text size='M' weight='bold'>
-              Import thingy
-            </Text>
-            <Text size='S'>Coming soon</Text>
-            <button className='bg-white' onClick={writeManager}>
-              Approve
-            </button>
-            <button className='bg-white' onClick={mint}>
-              Mint
-            </button>
+            <Text size='L'>Boost Factor</Text>
+            <div>
+              <Display size='M'>{`${boostFactor}x`}</Display>
+              <LeverageSlider
+                type='range'
+                list='boost-factor-labels'
+                min={BOOST_MIN}
+                max={BOOST_MAX}
+                step={1}
+                value={boostFactor}
+                onChange={(e) => setBoostFactor(parseInt(e.target.value) || 0)}
+              />
+              <StyledDatalist id='boost-factor-labels'>
+                {labels.map((label, i) => (
+                  <option key={i} value={i + 1} label={label}></option>
+                ))}
+              </StyledDatalist>
+            </div>
+            <div className='flex flex-col gap-1 w-full mt-auto'>
+              <Text size='M' weight='bold' className='w-full text-start'>
+                Summary
+              </Text>
+              <Text size='XS' color={SECONDARY_COLOR} className='w-full text-start overflow-hidden text-ellipsis'>
+                You're moving your liquidity from a Uniswap NFT to an Aloe NFT and are applying a{' '}
+                <strong>{boostFactor}x boost</strong>. As a result, you will earn swap fees {boostFactor}x faster, but
+                you will also be paying intest to Aloe lenders and risk liquidation. Liquidation thresholds are shown in
+                the graph to the left.
+              </Text>
+            </div>
+            <div className='w-full mt-8'>
+              <FilledStylizedButton
+                size='M'
+                fillWidth={true}
+                disabled={buttonState.disabled}
+                onClick={() => {
+                  if (state === ImportModalState.READY_TO_APPROVE) {
+                    writeManager?.();
+                  } else if (state === ImportModalState.READY_TO_MINT) {
+                    mint?.();
+                  }
+                }}
+              >
+                {buttonState.label}
+              </FilledStylizedButton>
+              <Text size='XS' color={TERTIARY_COLOR} className='w-full mt-2'>
```

Comment:
On the portfolio page, it looks like this text is usually left aligned, but here it's centered.